.github/workflows/push.yaml

```diff
@@ -15,6 +15,10 @@ env:
 
 jobs:
   build:
+    permissions:
```

Comment:
@jkroepke please explain why these permissions are needed? I don't think they are, we normally don't have to specify these permissions when pushing to ghcr

Reply:
I see. Normally, you need the permissions to push from GH Actions to ghcr. But you are using a PAT instead the actions token, which makes the additional permission request pointless.